src/pages/__tests__/Calculator.test.tsx

```diff
@@ -19,7 +19,7 @@ describe('calculator page', () => {
       },
     )
 
-    const interactionTypeLabelQuery = /Is this a specific activity or an ongoing relationship/i
+    const interactionTypeLabelQuery = /What do you want to learn/i
```

Comment:
I'm curious about the word "learn" here. It makes me think that the main point of this tool is to give me an education. (which... i get it... it is! haha.) I don't have a better suggestion yet. And maybe a lot of thought has already been put into the wording here, and if so, I'll follow your lead.

Possibilities:
* What do you want to know?
* What do you want to calculate? (I could see a case to be made that the word calculate might scare some people off)

Reply:
know works for me!